core/field_input.ts

```diff
@@ -457,22 +440,16 @@ export abstract class FieldInput<T extends InputTypes> extends Field<T> {
    *
    * @param e Keyboard event.
    */
-  protected onHtmlInputKeyDown_(e: Event) {
-    // AnyDuringMigration because:  Property 'keyCode' does not exist on type
-    // 'Event'.
-    if ((e as AnyDuringMigration).keyCode === KeyCodes.ENTER) {
+  protected onHtmlInputKeyDown_(e: KeyboardEvent) {
+    if (e.keyCode === KeyCodes.ENTER) {
       WidgetDiv.hide();
       dropDownDiv.hideWithoutAnimation();
-      // AnyDuringMigration because:  Property 'keyCode' does not exist on type
-      // 'Event'.
-    } else if ((e as AnyDuringMigration).keyCode === KeyCodes.ESC) {
+    } else if (e.keyCode === KeyCodes.ESC) {
       this.setValue(
           this.htmlInput_!.getAttribute('data-untyped-default-value'));
       WidgetDiv.hide();
       dropDownDiv.hideWithoutAnimation();
-      // AnyDuringMigration because:  Property 'keyCode' does not exist on type
-      // 'Event'.
-    } else if ((e as AnyDuringMigration).keyCode === KeyCodes.TAB) {
+    } else if (e.keyCode === KeyCodes.TAB) {
       WidgetDiv.hide();
       dropDownDiv.hideWithoutAnimation();
       // AnyDuringMigration because:  Property 'shiftKey' does not exist on type
```

Comment:
there is one more you can remove, on line 482

Reply:
Fixed!